src/guide/getting_started.md

```diff
@@ -110,15 +68,18 @@ class SDKListener: EventListener {
 }
 
 // Create the default config
-var config = defaultConfig(envType: EnvironmentType.production)
+let seed = try mnemonicToSeed(phrase: "<mnemonics words>")
+let inviteCode = "your invite code"
+let config = breez_sdk.defaultConfig(envType: EnvironmentType.production, apiKey: "", 
```

Comment:
I think we need to describe the difference between credentials & invite code.
That information was lost with:
> // register_node takes either greenlight credentials (certificate & key) or invite code. 
> // At this example we are using the invite code option.

Reply:
It was confusing since we had credentials before but now the user is not aware of credentials anymore so I think it is not confusing anymore.